examples/deployment-with-init/provider.tf

```diff
@@ -0,0 +1,14 @@
+terraform {
+  required_version = ">= 0.12"
+
+  required_providers {
+    ec = {
+      source = "elastic/ec"
+    }
+  }
+}
+
+provider "ec" {
+  apikey = "azFXa0pIUUJaeWRjMmUzVXpCOG06VUJ1NEtFREtSNE9BMThxeG9mM1A3dw=="
```

Comment:
@Kushmaro its a good idea to remove this

Reply:
Key is invalidated
but i'll remove this anyways :)